docs/articles/case-study-gdpr-it.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Case study: garantire il diritto all''oblio'
+created: '2020-11-13T16:43:39.086Z'
+modified: '2020-11-14T09:47:45.393Z'
+---
+
+# Case study: garantire il diritto all'oblio
+
+Quando progettiamo una nuova funzionalità per l'app IO, la più grande sfida che dobbiamo affrontare è quella di saper coniugare un'esperienza utente morbida con elevati requisiti di sicurezza, un'implementazione lineare con la complessità del contesto giuridico in cui IO si colloca rispetto al Cittadino. Questo richiede forte collaborazione tra sviluppatori, service designer, legali, esperti di sicurezza e di operations.
+
+Questo processo interdisciplinare è stato messo particolarmente alla prova per recepire le norme del GDPR e integrarne i principi all'interno dell'app IO. In particolare, ai Cittadini deve essere garantito il _diritto all'oblio_, cioè deve essere rispettata la loro eventuale decisione di rimuovere i propri dati personali che vengono conservati dall'app IO.
+
+In questo articolo vogliamo condividere le soluzioni tecniche messe in campo per garantire il diritto all'oblio del Cittadino. In particolare vorremmo condividere l'utilizzo di Azure Durable Functions, e di come abbiamo tratto veramente grande beneficio da questo strumento.
```

Comment:
`Cittadino` minuscolo?

Reply:
Secondo me no perché è un attore del sistema IO, quasi una keyword

Reply:
In questa frase non è un attore, è un semplice cittadino, nell'elenco degli use case invece si ci sta che sia maiuscolo